### PR TITLE
Robotiq Gripper Ignition Simulation

### PR DESCRIPTION
**Pull request by livanov93**
PR info:
- ignition simulation launch setup
- ignition simulation separate urdfs
- improve README
- add `.repos` file for easier dependency handling

### CODE REVIEW

**Comment by livanov93** [comment]
@corycrean ping for review

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+#### With Ignition simulation
+
+```
+ros2 launch robotiq_driver robotiq_simulation.launch.py
+```
+
+## Usage
+
+```
+ros2 action send_goal /robotiq_gripper_controller/gripper_cmd control_msgs/action/GripperCommand "{command: {position: 0.80, max_effort: 50.0}}"
```
Comment (corycrean) [inline README.md:49]: Should this work when not using real hardware?  I tried it with fake hardware and with Ignition simulation, and in both cases it said
```
Waiting for an action server to become available...
```
and the command never succeeded.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+```
+ros2 launch robotiq_driver robotiq_control.launch.py use_fake_hardware:=true
+```
+
+#### With Ignition simulation
+
+```
+ros2 launch robotiq_driver robotiq_simulation.launch.py
+```
```
Comment (corycrean) [inline README.md:44]: This looks fine in RViz, but not in Ignition:

https://user-images.githubusercontent.com/96085371/191117386-3f84c568-831b-45aa-b2e6-9bc65c6aed35.mp4

Comment (livanov93) [reply README.md:44]: @corycrean I would say the ignition is not installed properly

**Comment by livanov93** [comment]
Closing in favor of https://github.com/PickNikRobotics/ros2_robotiq_gripper/pull/8. #8 don't include all the needed repos and does not have usage instructions.

**Pull request closed without merging**